code/controllers/subsystem/shuttles/emergency.dm

```diff
@@ -105,11 +105,14 @@
 				priority_announce("Hostile enviroment detected. Departure has been postponed indefinitely pending conflict resolution.", null, 'sound/misc/notice1.ogg', "Priority")
 				mode = SHUTTLE_STRANDED
 			if(time_left <= 0 && !SSshuttle.emergencyNoEscape)
```

Comment:
Can't you just add 

if(time_left <=5 && !SSshuttle.emergencyNoEscape)
     play_cool_sound